frontend/src/components/regions/UserDetails/TeamsTable/Presentation.js

```diff
@@ -42,7 +42,7 @@ const TeamsTable = ({ teams, authUser }) => {
                 <TableCell>{team.name}</TableCell>
                 <TableCell>
                   {(authUser.isSuperuser ||
-                    authUser.permissions.teams[team.id]) && (
+                    authUser.teamMemberships[team]) && (
```

Comment:
Please don't fix the story in such a way as to break the actual frontend. 

We need the story to work AND we need Tilde to not break.

If you look at the redux store in the running application you will see that the authUser object looks lvery different from what you put there.